pkg/uefi/file.go

```diff
@@ -450,10 +451,12 @@ func NewFile(buf []byte) (*File, error) {
 
 	if ReadOnly {
 		f.buf = buf[:f.Header.ExtendedSize]
+		fmt.Fprintf(os.Stderr, "Readonly, buf size %v for %v\n  ( %v )\n\n", f.Header.ExtendedSize, f.Type, f.Header.GUID)
```

Comment:
Please do not write to `os.Stderr` directly. There is package `github.com/linuxboot/fiano/log` for warnings and errors. We may consider adding more levels if required, but anyway :)